docs/src/bpmn-workflows/timer-events/timer-events.md

```diff
@@ -22,23 +22,30 @@ When an intermediate timer catch event is entered then a corresponding timer is
 
 An interrupting timer boundary event must have a **time duration** definition. When the corresponding timer is triggered then the activity gets terminated. Interrupting timer boundary events is often used to model timeouts, for example, canceling the processing after 5 minutes and do something else.
 
-An non-interrupting timer boundary event must have either a **time duration or time cycle** definition. When the activity is entered then it schedules a corresponding timer. If the timer is triggered and it is defined as time cycle with repetitions > 0 then it schedules the timer again until the defined number of repetitions is reached. Non-interrupting timer boundary events is often used to model notifications, for example, contacting the support if the processing takes longer than one hour. 
+An non-interrupting timer boundary event must have either a **time duration or time cycle** definition. When the activity is entered then it schedules a corresponding timer. If the timer is triggered and it is defined as time cycle with repetitions > 0 then it schedules the timer again until the defined number of repetitions is reached. Non-interrupting timer boundary events is often used to model notifications, for example, contacting the support if the processing takes longer than one hour.
 
 ## Timers
 
-Timers must be defined by providing either a date, a duration, or a cycle. 
+Timers must be defined by providing either a date, a duration, or a cycle.
+
+A timer can be defined either as a static value (e.g. `PT3D`) or as an [expression](/reference/expressions.html). There are two common ways for using an expression:
+
+* [access a variable](/reference/expressions.html#access-variables) (e.g. `= remainingTime`)
+* [using temporal values](/reference/expressions.html#temporal-expressions) (e.g. `= date and time(expirationDate) - date and time(creationDate)`)
+
+The expression is evaluated on activating the timer catch event. It must result either in a `string` that has the same ISO 8601 format as the static value or an equivalent temporal value (i.e. a date-time, a duration, or a cycle).
```

Comment:
This timers paragraphs is not necessarily only for timer catch events. I think we should also mention when a timer start event expression is evaluated.

Reply:
Good point.